sql/moz-fx-data-shared-prod/telemetry_derived/unified_metrics_v1/checks.sql

```diff
@@ -30,7 +30,6 @@ WHERE submission_date = @submission_date;
   "days_since_seen",
   "is_new_profile",
   "normalized_os",
-  "normalized_os_version",
```

Comment:
This is null in some cases

Reply:
I'm fine with removing this one from the list.